### ISSUE TITLE
Ripple needs the children prop too

### ISSUE THREAD

**Issue by meluskyc** [telerik/kendo-react#1321]
## I'm submitting a...
* Bug report

## Current behavior
`Type '{ children: Element; key: number; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Ripple> & Pick<Readonly<RippleProps>, keyof RippleProps> & Pick<...> 
& Pick<...>'.
  Property 'children' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Ripple> & Pick<Readonly<RippleProps>, keyof RippleProps> & Pick<...> & Pick<...>'.`

## Expected behavior
No type error

## Minimal reproduction of the problem with instructions
Add a Ripple component with some children

## Environment

Package versions:
```
+-- @progress/kendo-data-query@1.5.6
+-- @progress/kendo-drawing@1.16.3
+-- @progress/kendo-licensing@1.2.2
+-- @progress/kendo-react-animation@5.3.0
+-- @progress/kendo-react-buttons@5.3.0
+-- @progress/kendo-react-common@5.3.0
+-- @progress/kendo-react-data-tools@5.3.0
+-- @progress/kendo-react-dateinputs@5.3.0
+-- @progress/kendo-react-dialogs@5.3.0
+-- @progress/kendo-react-dropdowns@5.3.0
+-- @progress/kendo-react-excel-export@5.3.0
+-- @progress/kendo-react-form@5.3.0
+-- @progress/kendo-react-grid@5.3.0
+-- @progress/kendo-react-indicators@5.3.0
+-- @progress/kendo-react-inputs@5.3.0
+-- @progress/kendo-react-intl@5.3.0
+-- @progress/kendo-react-layout@5.3.0
+-- @progress/kendo-react-listbox@5.3.0
+-- @progress/kendo-react-listview@5.3.0
+-- @progress/kendo-react-notification@5.3.0
+-- @progress/kendo-react-progressbars@5.3.0
+-- @progress/kendo-react-ripple@5.3.0
+-- @progress/kendo-react-tooltip@5.3.0
+-- @progress/kendo-react-treelist@5.3.0
+-- @progress/kendo-react-treeview@5.3.0
+-- @progress/kendo-theme-default@5.4.1
```

Browser:

System:
- TypeScript version: Version 4.7.2
- Node version: v16.14.0 npm 8.3.1
- Platform:  Windows

**Comment by nstoychev** [comment]
@meluskyc - The Ripple component has been already updated with `children` prop and the fix is not yet officially published. In order to check it, see the following article about obtaining our development builds - https://www.telerik.com/kendo-react-ui/components/installation/development-builds/.

**Comment by meluskyc** [comment]
Fixed in 5.4. Thanks!
